requirements/base.in

```diff
@@ -14,3 +14,4 @@ pyjwkest
 edx-opaque-keys[django]
 django-filter
 jsonfield2
+django-config-models>=1.0.0         # Configuration models for Django allowing config management with auditing
```

Comment:
This line is copy-pasted form edx-platform: https://github.com/edx/edx-platform/blob/0be941b224f2bd0183b21be9a7b29c2666cc78ee/requirements/edx/base.in#L47

It is pinned there, and as such I pinned it here so that the version installed should be the same.

Reply:
> This line is copy-pasted form edx-platform: https://github.com/edx/edx-platform/blob/0be941b224f2bd0183b21be9a7b29c2666cc78ee/requirements/edx/base.in#L47
> 
> It is pinned there, and as such I pinned it here so that the version installed should be the same.

you are right, but OEP-18 describes something as follows. I think that needs to be updated in edx as well but because we are adding a requirement with constraint in this we should follow OEP-18. 
https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html#imposing-constraints-on-dependencies

Reply:
Sure, will do!

Reply:
Thanks @xitij2000